src/ariel-os-rt/src/lib.rs

```diff
@@ -66,6 +72,27 @@ mod isr_stack {
         "#,
         size = const ISR_STACKSIZE
     );
+
+    pub fn limits() -> (usize, usize) {
+        // ISR stack
+        // TODO: multicore!
+        unsafe extern "C" {
+            static _stack_bottom: u32;
+            static _stack_start: u32;
```

Comment:
I understand that these come from here:

https://github.com/ariel-os/ariel-os/blob/39b7746e2503628b44f8dc247cb0f9ceb6a7bbcf/src/ariel-os-rt/isr_stack.ld.in#L16-L17

But I think these should be `usize`:

```suggestion
            static _stack_bottom: usize;
            static _stack_start: usize;
```

Reply:
type is irrelevant, linker passes values as *addresses* of symbols.

Reply:
Indeed!